server/handlers/default.go

```diff
@@ -145,3 +146,84 @@ func DeleteHandler(ctx context.Context, w http.ResponseWriter, r *http.Request)
 	}
 	return nil
 }
+
+func GetTimestampHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) *errors.HTTPError {
+	s := ctx.Value("metaStore")
+	store, ok := s.(storage.MetaStore)
+	if !ok {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
+			Err:        fmt.Errorf("Version store not configured"),
+		}
+	}
+	sign := ctx.Value("signer")
+	signer, ok := sign.(*signed.Signer)
+	if !ok {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
+			Err:        fmt.Errorf("Signer not configured"),
+		}
+	}
+
+	vars := mux.Vars(r)
+	gun := vars["imageName"]
+
+	out, err := timestamp.GetOrCreateTimestamp(gun, store, signer)
+	if err != nil {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
```

Comment:
Placeholder error code. One of my cleanup tasks is to create a set of error codes.